src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2009,7 +2009,7 @@ private BoundExpression BindSuppressNullableWarningExpression(PostfixUnaryExpres
             if ((object)type != null)
             {
                 // Report diagnostic if the type consists of no reference types.
-                if ((object)type.VisitType((t, a, c) => t.IsErrorType() || t.IsReferenceType, (object)null, canDigThroughNullable: true) == null)
+                if ((object)type.VisitType((t, a, c) => t.IsErrorType() || t.IsReferenceType || t.IsUnconstrainedTypeParameter(), (object)null, canDigThroughNullable: true) == null)
```

Comment:
From what I can tell, the only test that covers this (produce the diagnostic `ERR_NotNullableOperatorNotReferenceType` on unconstrained `T`) is skipped.
Is there a blocking issue preventing to test even one case?